master/buildbot/changes/gerritchangesource.py

```diff
@@ -257,7 +257,7 @@ def __init__(self, change_source):
         @defer.inlineCallbacks
         def outReceived(self, data):
             """Do line buffering."""
-            self.data += data
+            self.data += bytes2NativeString(data)
```

Comment:
This operates on a JSON-encoded string, so str seems more appropriate than bytes. You can also see tests like `master/buildbot/test/unit/test_changes_gerritchangesource.py:146:` that pass `json.dumps(...)`, suggesting that the expected parameter for `lineReceived` is a str and not bytes.